pkg/operator/operator.go

```diff
@@ -55,10 +58,11 @@ func ValidateTriggerDeployments(cs *clients.Clients, rnames utils.ResourceNames)
 		config.TriggerControllerName, config.TriggerWebhookName)
 }
 
-func ValidateOperatorInstallStatus(cs *clients.Clients, rnames utils.ResourceNames) {
-	log.Printf("Waiting for operator to be up and running....\n")
-	EnsureTektonConfigStatusInstalled(cs.TektonConfig(), rnames)
-	log.Printf("Operator is up\n")
+func ValidateOperatorInstallStatus() {
+	operatorVersion := cmd.MustSucceed("tkn", "version", "--component", "operator").Stdout()
+	if strings.Contains(operatorVersion, "unknown"){
```

Comment:
@VeereshAradhya The main motivation for this PR is that we still have special roles on our testing clusters https://issues.redhat.com/browse/SRVKP-1744 but it seem to be unnecessary (at least for 1.10, we would need to test older releases).

The old implementation of this function waits until TektonConfig reports `InstallSucceeded=true`, on the other hand if the operator is not being installed, it will wait too long. New implementation fails immediately but on the other hand provides false negative e.g. when you remove the TektonConfig instance but not subscription.

This is the result of `oc get tektonconfig && tkn version --component operator` right after I click install in UI:

```
NAME     VERSION   READY   REASON
config
v0.65.1
```

IMHO we could use both... first run `tkn version`, then check status of TektonConfig. That way it would wait until the operator installation finishes but would fail fast if the operator is not installed at all. What do you think?